packages/gatsby/src/utils/test-require-error.ts

```diff
@@ -16,6 +16,6 @@ export const testRequireError = (moduleName: string, err: any): boolean => {
       `\\$&`
     )}`
   )
-  const firstLine = err.toString().split(`\n`)[0]
-  return regex.test(firstLine)
+  const [firstLine] = err.toString().split(`\n`)
+  return regex.test(firstLine.replace(`\\\\`, `\\`))
```

Comment:
Is it supposed to replace just one or any?
```suggestion
  return regex.test(firstLine.replace(/\\\\/g, `\\`))
```

Reply:
If it is indeed any, maybe a test would help to confirm :shrug:

Reply:
@pvdz I already made a PR for this on @wesbos's repo which includes the `/g` fix and a test :slightly_smiling_face: 

https://github.com/wesbos/gatsby/pull/3

Reply:
Let me know if that test looks ok so I can update before Wes merged